client/my-sites/plans/controller.jsx

```diff
@@ -89,11 +89,12 @@ export default {
 			productsList = require( 'lib/products-list' )(),
 			analyticsPageTitle = 'Plans > Compare',
 			site = sites.getSelectedSite(),
+			siteDomain = context.params.domain,
 			basePath = route.sectionify( context.path );
 		let baseAnalyticsPath;
 
 		if ( config.isEnabled( 'manage/plan-features' ) ) {
-			return page.redirect( '/plans/features' );
+			return page.redirect( `/plans/features/${ siteDomain || '' }` );
```

Comment:
minor, but why not set the else value when we assign the value initially? I think it's cleaner there.